get_display_style should not be used in add time tracking popup window

to reproduce:
1. click hide on issue details screen of time tracking
2. click show it again
3. click hide once again to set cookie being in state "do not show"
4. click add time tracking. you'll get add time tracking popup with just footer

--- eventum/templates/en/add_phone_entry.tpl.html~	2006-02-09 19:19:28.000000000 +0200
+++ eventum/templates/en/add_phone_entry.tpl.html	2006-02-10 13:40:01.641181506 +0200
@@ -65,10 +65,10 @@
           <td>
             <table bgcolor="#FFFFFF" width="100%" cellspacing="1" cellpadding="2">
               {if $current_role > $roles.customer}
-              <tr id="phone_support2" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td colspan="2" class="default"><b>Record Phone Call:</b></td>
               </tr>
-              <tr id="phone_support3" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Date of Call:</b></td>
                 <td bgcolor="{$light_color}" width="100%">
                   {html_select_date start_year='-1' day_value_format='%02d' field_array='date' prefix='' all_extra=' class="default"'}&nbsp;
@@ -76,7 +76,7 @@
                   <a href="javascript:void(null);" onClick="javascript:updateTimeFields('add_phone_form', 'date[Year]', 'date[Month]', 'date[Day]', 'date[Hour]', 'date[Minute]');"><img src="images/icons/refresh.gif" border="0"></a>
                 </td>
               </tr>
-              <tr id="phone_support4" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Reason:</b></td>
                 <td bgcolor="{$light_color}" width="100%">
                   <select name="phone_category" class="default">
@@ -85,21 +85,21 @@
                   {include file="error_icon.tpl.html" field="phone_category"}
                 </td>
               </tr>
-              <tr id="phone_support5" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Call From:</b></td>
                 <td bgcolor="{$light_color}" width="100%">
                   <input type="text" class="default" name="from_lname" value="last name" onFocus="javascript:if (this.value == 'last name') this.value='';"><span class="default">,</span>
                   <input type="text" class="default" name="from_fname" value="first name" onFocus="javascript:if (this.value == 'first name') this.value='';">
                 </td>
               </tr>
-              <tr id="phone_support6" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Call To:</b></td>
                 <td bgcolor="{$light_color}" width="100%">
                   <input type="text" class="default" name="to_lname" value="last name" onFocus="javascript:if (this.value == 'last name') this.value='';"><span class="default">,</span>
                   <input type="text" class="default" name="to_fname" value="first name" onFocus="javascript:if (this.value == 'first name') this.value='';">
                 </td>
               </tr>
-              <tr id="phone_support7" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Type:</b></td>
                 <td bgcolor="{$light_color}" width="100%">
                   <select class="default" name="type">
@@ -108,7 +108,7 @@
                   </select>
                 </td>
               </tr>
-              <tr id="phone_support8" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Customer Phone Number:</b></td>
                 <td bgcolor="{$light_color}" width="100%">
                   <input type="text" size="20" maxlength="32" name="phone_number" class="default">
@@ -122,18 +122,18 @@
                   </select>
                 </td>
               </tr>
-              <tr id="phone_support9" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Time Spent:</b></td>
                 <td bgcolor="{$light_color}" width="100%"><input type="text" size="5" name="call_length" class="default"> {include file="error_icon.tpl.html" field="call_length"} <span class="small_default"><i>(in minutes)</i></span></td>
               </tr>
-              <tr id="phone_support10" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td bgcolor="{$internal_color}" class="default_white" width="190" nowrap><b>Description:</b></td>
                 <td bgcolor="{$light_color}">
                   <textarea name="description" rows="8" style="width: 97%"></textarea>
                   {include file="error_icon.tpl.html" field="description"}
                 </td>
               </tr>
-              <tr id="phone_support11" {get_display_style element_name="phone_support"}>
+              <tr>
                 <td colspan="2" bgcolor="{$internal_color}" align="center">
                   <input type="submit" value="Save Phone Call" class="button">
                 </td>
--- eventum/templates/en/mail_queue.tpl.html~	2006-02-10 16:26:46.000000000 +0200
+++ eventum/templates/en/mail_queue.tpl.html	2006-02-10 18:35:28.462720077 +0200
@@ -12,7 +12,7 @@
         <b>Mail Queue for Issue #{$issue_id}</b>
       </td>
     </tr>
-    <tr id="drafts1" {get_display_style element_name="drafts"}>
+    <tr>
       <td>
         <table width="100%" cellpadding="2" cellspacing="1">
           <tr bgcolor="{$cell_color}">
